File: Ejercicios iOS/Personita-Objective-C/PersonitaApp/Personita.m
//
//  Personita.m
//  PersonitaApp
//
//  Created by Virtualizacion02 on 24/08/23.
//

#import "Personita.h"

@interface Personita ()

@end

@implementation Personita

-(id) initWithNombre: (NSString *) nombre Edad: (int) edad{
    
    self=[super init];
    _nombre = nombre;
    _edad = edad;
    
    return self;
}

-(NSString *) saludar{
    NSString *saludo;
    
    saludo = [NSString stringWithFormat:@"Hola soy %@ y mi edad es %d", _nombre, _edad];
    
    return saludo;
}

@end

File: Ejercicios iOS/Personita-Swift/PersonitaAppSwift/Personita.swift
//
//  Personita.swift
//  PersonitaAppSwift
//
//  Created by Virtualizacion02 on 24/08/23.
//

import UIKit

class Personita{

    let nombre:String
    let edad: Int
    
    init(nombre: String, edad: Int) {
        self.nombre = nombre
        self.edad = edad
    }
    
    func saludar()->String{
        var saludo:String = "Hola \(nombre) tienes \(edad)"
        return saludo
    }
}